frontend/css/map.css

```diff
@@ -0,0 +1,105 @@
+body {
+    margin: 0 0 0 0;
+    padding: 0 0 0 0;
+    overflow-y: hidden;
+}
+/* HEADER BOX */
+header {
+    background-color: rgb(167, 60, 113);
+    filter: drop-shadow(0px 10px 10px rgba(0,0,0,.15));
+    width: 100vw;
+    height: 11vh;
+    position: absolute;
+    margin: 0 0 0 0;
+    z-index: 2;
+}
+#bus-name {
+    color: white;
+    margin: 0 0 0 25%;
+    padding: 0 0 0 0;
+    font-family: 'Lato';
+    font-size: 22px;
+    height: 11vh;
+    line-height: 11vh;
+    text-align: center;
+    width: 50vw;
+    filter: none;
+    position: absolute;
+    z-index: 3;
+}
+
+/* MAP CONTENT */
+#map {
+    position: absolute; 
+    top: 11vh;
+    bottom: 0; 
+    width: 100%;
+    height: 79vh;
+    z-index: 0;
+}
```

Comment:
once we get rid of the footer, we can now make the map take up a larger portion of our page by adjusting it's `height` to `89vh`